### PR TITLE
Document Coding Convention

### LINKED ISSUE #5: Document Coding Convention
- on `docs/convention.md`

### PR DESCRIPTION
**Pull request by hataehyeok**
This guidelines include
- Code Style
- Git Strategy

closes #5

### CODE REVIEW

File: docs/convention.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  - Version: `3.7.14`
+  - Configuration:
+
+    ```properties
+    version = 3.7.14
+    runner.dialect=scala213source3
+    ```
+
+- **scalafix**:
+  - Adhere to the default configurations of `scalafmt` and `scalafix`.
```
Comment (betarixm) [inline docs/convention.md:26]:
```suggestion
  - Adhere to the default configurations of `scalafix`.
```

File: docs/convention.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- **main**:
+  - **WARNING**: DO NOT PUSH DIRECTLY TO THIS BRANCH.
+  - Only merge through PRs with at least one review.
+  - Only merges from `develop`.
+- **develop**:
+  - You can commit freely but it's recommended to branch off for work as described below.
+  - Branch naming conventions:
+    - Features: `feat/kebab-case`
+    - Fixes: `fix/kebab-case`
+    - Chores: `chore/kebab-case`
```
[the remark this answers is not in the document]
Comment (hataehyeok) [reply docs/convention.md:56]: Fixed

File: docs/convention.md (hunk 1 above)
Comment (betarixm) [reply docs/convention.md]: e7ff7e4908b7dc692800cccd9f9ca09fdd5a2c06

**Review by betarixm — APPROVED** [review]
lgtm

**Review by leejiwon1125 — APPROVED** [review]
LGTM!

**Pull request merged**